packages/ui/src/hooks/localization.ts

```diff
@@ -0,0 +1,37 @@
+const getLanguage = () => {
```

Comment:
attempts to use browser locale, defaults to english

Reply:
Since we dont have any translation system in place, I guess this will just affect numbers, dates... And hope they work nice with English (I imagine giving a German month in an English text as bad experience)